src/main/resources/portal.properties.EXAMPLE

```diff
@@ -391,3 +391,6 @@ persistence.cache_type=no-cache
 
 # Feature flag for treatment groups
 # enable_treatment_groups=false
+
+# Set comparison categorical "NA" value (not case sensitive)
+comparison.categorical_na_value=NA,unknown
```

Comment:
@leexgh default should be commented out - that is to avoid us from being able to change the default value when people upgrade to a new version of the portal